api/routes.go

```diff
@@ -180,18 +110,20 @@ func (api *API) RoutesDeleteHandler(w http.ResponseWriter, r *http.Request) {
 	if api.cfg.Authenticate && !cas.IsAuthenticated(r) {
 		return
 	}
+
 	id := chi.URLParam(r, "id")
 	log.Debugf("deleting", id)
 	err := api.db.DeleteRoute(id)
+
 	// Error handling
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
 }
 
 type sched struct {
-	Times []model.WeekTime `json:times`
-	ID    string           `json:id`
+	Times []model.Time `json:times`
+	ID    string       `json:id`
```

Comment:
These struct tags need quotation marks around the names https://travis-ci.org/wtg/shuttletracker/jobs/357929397#L549